frb/frb.py

```diff
@@ -13,7 +13,7 @@
 
 from astropy.coordinates import SkyCoord
 from astropy import units
-from astropy.cosmology import Planck15
+from astropy.cosmology import Planck18
```

Comment:
I would import this with an alias. e.g. `import  Planck18 as frb_cosmo`. That way, if we change the cosmology in the future, we only need to change it in the import statement.

Reply:
In fact, it might be a good idea to import `frb_cosmo` from `defs.py` everywhere in the repo. That way, we can be sure that it's truly uniform.